packages/cli/src/middleware/enforceNodeVersionConfig.js

```diff
@@ -0,0 +1,39 @@
+import boxen from 'boxen'
+import semver from 'semver'
+
+import { getConfig } from '@redwoodjs/project-config'
+
+export async function enforceNodeVersionConfig() {
```

Comment:
@Josh-Walker-GM sorry to insert myself into this conversation, I'm a little bit scared of doing checks on the CLI on every run - could I suggest that we wrap the whole thing in a try catch? 

This way if anything fails in the logic, for example cooercing the semver version (which happens sometimes, for example with the experimental releases) - it won't stop someone from using rw.

Reply:
Yeah that's certainly something we should to do, happy to.